src/router/handlers/admin.js

```diff
@@ -0,0 +1,356 @@
+ */
+let createOrganization = async (req, res) => {
+
+};
+
+/**
+ * @swagger
+ * /org/:
+ *  put:
+ *    description: Use to update an organization
+ *    parameters:
+ *      - name: ha-api-token
+ *        in: header
+ *        required: true
+ *        schema:
+ *          type: string
+ *          format: uuid
+ *      - name: organization
+ *        in: body
+ *        required: true
+ *        schema:
+ *          type: object
+ *    responses:
+ *      '200':
+ *        description: Success
+ *      '400':
+ *        description: 'Invalid syntax'
+ *      '401':
+ *        description: 'JWT not found in header'
+ *      '403':
+ *        description: 'JWT does not have admin privileges'
+ *      '404':
+ *        description: 'Not found'
+ */
+let updateOrganization = async (req, res) => {
+
+};
+
+/**
+ * @swagger
+ * /hacks/:
+ *  post:
+ *    description: Use to create a hackathon
+ *    parameters:
+ *      - name: ha-api-token
+ *        in: header
+ *        required: true
+ *        schema:
+ *          type: string
+ *          format: uuid
+ *      - name: hackathon
+ *        in: body
+ *        required: true
+ *        schema:
+ *          type: object
+ *    responses:
+ *      '201':
+ *        description: 'Success'
+ *      '400':
+ *        description: 'Invalid syntax'
+ *      '401':
+ *        description: 'JWT not found in header'
+ *      '403':
+ *        description: 'JWT does not have admin privileges'
+ */
+let createHackathon = async (req, res) => {
+
+};
+
+
+/**
+ * @swagger
+ * /hacks/:hid/:
+ *  put:
+ *    description: Use to update a hackathon
+ *    parameters:
+ *      - name: ha-api-token
+ *        in: header
+ *        required: true
+ *        schema:
+ *          type: string
+ *          format: uuid
+ *      - name: hackathon
+ *        in: body
+ *        required: true
+ *        schema:
+ *          type: object
+ *    responses:
+ *      '200':
+ *        description: Success
+ *      '400':
+ *        description: 'Invalid syntax'
+ *      '401':
+ *        description: 'JWT not found in header'
+ *      '403':
+ *        description: 'JWT does not have admin privileges'
+ *      '404':
+ *        description: 'Not found'
+ */
+let updateHackathon = async (req, res) => {
+
+};
+
+/**
+ * @swagger
+ * /hacks/:hid/:
+ *  delete:
+ *    description: Use to delete a hackathon
+ *    parameters:
+ *      - name: ha-api-token
+ *        in: header
+ *        required: true
+ *        schema:
+ *          type: string
+ *          format: uuid
+ *    responses:
+ *      '200':
+ *        description: Success
+ *      '400':
+ *        description: 'Invalid syntax'
+ *      '401':
+ *        description: 'JWT not found in header'
+ *      '403':
+ *        description: 'JWT does not have admin privileges'
+ *      '404':
+ *        description: 'Not found'
+ */
+let deleteHackathon = async (req, res) => {
+
+};
+
+/**
+ * @swagger
+ * /hacks/details/:
```

Comment:
Should this be `/hacks/:hid/details/:` to specify which hackathon's details are being created? It is included for later ones.

Reply:
Correct!